docs/compass/02-02-components.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Compass components
+type: Architecture
+---
+
+Compass consists of a set of components that allow you to extend, customize and integrate your applications.
+
+![Components](./assets/components.svg)
+
+## Cockpit
+
```

Comment:
Perhaps it is better to list them instead of creating separate sections out of them? Then, it would be consistent with docs in other topics, like https://kyma-project.io/docs/components/service-catalog/#details-resources or https://kyma-project.io/docs/components/asset-store/#architecture-architecture-resources?

Reply:
I based the layout of this doc on the one we currently have in the Application Connector:
https://kyma-project.io/docs/components/application-connector/#architecture-application-connector-components

TBH, I prefer sections to the list (moreover, in the SC documentation we list resources, not components, and that makes a difference). If we mention a given component in any other doc, it will be possible to link to the section devoted to this component in this document. In case of the list, it's not possible.